openassessment/workflow/team_api.py

```diff
@@ -0,0 +1,139 @@
+"""
+Functions that largely mirror the workflow functions specified
+in `workflow.api`, but specifically for handling team submissions.
+"""
+import logging
+
+from openassessment.workflow.errors import (
+    AssessmentWorkflowInternalError,
+    AssessmentWorkflowRequestError,
+    AssessmentWorkflowNotFoundError
+)
+from openassessment.workflow.models import TeamAssessmentWorkflow
+from openassessment.workflow.serializers import TeamAssessmentWorkflowSerializer
+
+
+logger = logging.getLogger('openassessment.workflow.models')  # pylint: disable=invalid-name
```

Comment:
It's customary to do `logger = logging.getLogger(__name__)`, which helps avoid copypasta :)